pom.xml

```diff
@@ -848,6 +848,10 @@
         <artifactId>hbase-common</artifactId>
         <version>${hbase10.version}</version>
         <exclusions>
+          <exclusion>
```

Comment:
so we dont' need to have any replacement? excluding it will just work ?

Reply:
yes. We don't need the tools jar to build CDAP.

Reply:
got it, thank you :)